wagtail/admin/templates/wagtailadmin/shared/header.html

```diff
@@ -47,7 +47,7 @@ <h1 class="w-header__title" id="header-title">
             {% if search_url %}
                 <form
                     class="col search-form"
-                    action="{% url search_url %}{% if query_parameters %}?{{ query_parameters }}{% endif %}"
+                    action="{% resolve_url search_url %}{% if query_parameters %}?{{ query_parameters }}{% endif %}"
```

Comment:
I don’t understand this change?

Reply:
This was unnecessary but I forgot to drop the commit